drivers/storage/class/disk/disk.inf

```diff
@@ -200,6 +200,19 @@ mylex_array.DeviceDesc = "Urządzenie dyskowe Mylex RAID"
 
 VHD_Generic_FriendlyName = "Dysk wirtualny firmy Microsoft"
 
+[Strings.0416]
+DiskClassName="Unidade de disco"
+GenManufacturer="(Unidade de disco padrão)"
+
+disk_devdesc="Unidade de disco"
+opticaldisk_devdesc="Unidade de disco ótica"
```

Comment:
"ótica" of the ear or referring to it. (from the Greek otikós)
"óptica" referring to vision. (from the Greek optikós)

Reply:
"óptica" referring to vision. (from the Greek optikós)
in this word the 'p' is perceptible and very audible to distinguish it from "ótica" referring to the ear.